project/Dependencies.scala

```diff
@@ -14,18 +14,17 @@ object Dependencies {
   val nailgunCommit = "a2520c1e"
 
   // Keep in sync in BloopComponentCompiler
-  val zincVersion = "1.7.2"
+  val zincVersion = "1.8.0"
 
   val bspVersion = "2.1.0-M3"
 
   val scalazVersion = "7.2.35"
-  val scalaXmlVersion = "1.2.0"
-  val lmVersion = "1.1.5"
+  val lmVersion = "1.8.0"
   val configDirsVersion = "26"
   val caseAppVersion = "2.0.6"
   val sourcecodeVersion = "0.3.0"
   val sbtTestInterfaceVersion = "1.0"
-  val sbtTestAgentVersion = "1.4.4"
+  val sbtTestAgentVersion = "1.8.0"
```

Comment:
Actually missed this before I said LGTM, but let's also make sure to bump this in `TestInternals`.

https://github.com/scalacenter/bloop/blob/main/frontend/src/main/scala/bloop/testing/TestInternals.scala#L49

Reply:
Done, thanks for noticing it!